### PR TITLE
Fix Attestation Readme Issue #28411

### LINKED ISSUE #28411: Attestation Readme Issue
1.
**Section** [link](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/attestation/azure-security-attestation#examples)
image

**Reason:**
The example title above and below is inconsistent.

**Suggestion:**
1. Change text to `Create a synchronous attestation client`
2. Change text to `Retrieve Token Certificates`
3. Change text to `Create a synchronous administrative client`
4. Change text to `Retrieve current attestation policy for OpenEnclave`
5. Change text to `List attestation signing certificates`
6. Change text to `Add attestation signing certificate`
7. Add link `- [Remove attestation signing certificate](#remove-attestation-signing-certificate)`

@joshfree, @lmazuel and @LarryOsterman for notification.

### PR DESCRIPTION
**Pull request by v-hongli1**
Fixes #28411

@joshfree, @lmazuel and @LarryOsterman for notification.

### CODE REVIEW

File: sdk/attestation/azure-security-attestation/README.md [hunk 1]
```diff
@@ -127,15 +127,16 @@ clients to add, remove or enumerate the policy management certificates.
 
 ## Examples
 
-- [Instantiate a synchronous attestation client](#create-a-synchronous-attestation-client)
-- [Retrieve token validation certificates](#retrieve-token-certificates)
+- [Create a synchronous attestation client](#create-a-synchronous-attestation-client)
+- [Retrieve Token Certificates](#retrieve-token-certificates)
```
Comment (LarryOsterman) [inline sdk/attestation/azure-security-attestation/README.md:131]: Change the anchor, not the link for `Retrieve Token Certificates`. The old name is better.

File: sdk/attestation/azure-security-attestation/README.md [hunk 2]
```diff
@@ -127,15 +127,16 @@ clients to add, remove or enumerate the policy management certificates.
...
 - [Attest an SGX enclave](#attest-an-sgx-enclave)
-- [Instantiate a synchronous administrative client](#create-a-synchronous-administrative-client)
-- [Get attestation policy](#retrieve-current-attestation-policy-for-openenclave)
-- [Set unsigned attestation policy](#set-unsigned-attestation-policy-aad-clients-only)
+- [Create a synchronous administrative client](#create-a-synchronous-administrative-client)
+- [Retrieve current attestation policy for OpenEnclave](#retrieve-current-attestation-policy-for-openenclave)
+- [Set unsigned attestation policy (AAD clients only)](#set-unsigned-attestation-policy-aad-clients-only)
```
Comment (LarryOsterman) [inline sdk/attestation/azure-security-attestation/README.md:135]: Change the anchor, not the link for `Set unsigned attestation policy`

File: sdk/attestation/azure-security-attestation/README.md [hunk 3]
```diff
@@ -127,15 +127,16 @@ clients to add, remove or enumerate the policy management certificates.
...
 - [Set signed attestation policy](#set-signed-attestation-policy)
-- [List policy management certificates](#list-attestation-signing-certificates)
-- [Add policy management certificate](#add-attestation-signing-certificate)
+- [List attestation signing certificates](#list-attestation-signing-certificates)
```
Comment (LarryOsterman) [inline sdk/attestation/azure-security-attestation/README.md:137]: Change the anchor, not the link for `List policy management certificates`.

Bonus: You might consider changing the name from `policy management certificates` to `Isolated Mode management certificates` - the name `policy management certificates` does not fully encapsulate what these certificates are used for.

File: sdk/attestation/azure-security-attestation/README.md [hunk 4]
```diff
@@ -127,15 +127,16 @@ clients to add, remove or enumerate the policy management certificates.
...
+- [Add attestation signing certificate](#add-attestation-signing-certificate)
```
Comment (LarryOsterman) [inline sdk/attestation/azure-security-attestation/README.md:138]: Change the anchor, not the link for `Add policy management certificate`

**Review by LarryOsterman — CHANGES_REQUESTED** [review]

File: sdk/attestation/azure-security-attestation/README.md [hunk 5]
```diff
@@ -127,15 +127,16 @@ clients to add, remove or enumerate the policy management certificates.
...
+- [Remove attestation signing certificate](#remove-attestation-signing-certificate)
```
[the remark this answers is not in the document]
Comment (hongli750210) [reply sdk/attestation/azure-security-attestation/README.md:139]: Rollbacked link name to old name and changed the anchor.

**Review by LarryOsterman — APPROVED** [review]

**Comment by v-xuto** [comment]
@LarryOsterman Please help merge this PR. Thanks a lot.

**Pull request merged**